pytest_sftpserver/core.py

```diff
@@ -0,0 +1,19 @@
+from pytest_sftpserver.sftp.server import SFTPServer
```

Comment:
I don't particularly like `core` as the module name. Maybe instead call it `decorator`.

It could optionally be re-exported from the root package as well.